src/app/course-page/courses-routing.module.ts

```diff
@@ -0,0 +1,31 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+import { CoursePageComponent } from './course-page.component';
+import { CourseAddComponent } from '../course-add/course-add.component';
+
+const routes: Routes = [
+  {
```

Comment:
Do you need those routes? As I see routers are already defined in app-routing.module